tools/buildroot/generate-kernel-initrd.sh

```diff
@@ -0,0 +1,123 @@
+#! /bin/bash
```

Comment:
It would be best to convert that to Python. The only reason it's a shell script is historical and due to me not pushing back enough back in the days :)

You can use `argparse` from the stdlib for the CLI, but there are some others like `click` which is quite popular and possibly require less boilerplate

Reply:
This is really simple shell script. If we ignore empty lines, copyright, global variables, and parameter parsing stuff, then it's in fact less than 10 lines which do trivial jobs (literally git clone/checkout, mkdir, cp, make). And I think a shell script is a good fit for this kind of basic work.